test/spec/znkExercise/znkExerciseDrv.spec.js

```diff
@@ -695,15 +695,15 @@ describe('testing directive "znkExerciseDrv":', function () {
         expect(content.hasClass('done-btn-show')).toBeFalsy();
     });
 
-    it('given view mode is review and initForceDoneBtnDisplay is true then only done-btn-show class should be added',function(){
+    it('given view mode is review and initForceDoneBtnDisplay is true then it still should hide the done button',function(){
```

Comment:
If tomorrow we will need to show done button in review, how we will implement it ?